pkg/service/http_service.go

```diff
@@ -599,26 +616,201 @@ func (s *HttpService) listJobsHandler(w http.ResponseWriter, r *http.Request) {
 	var jobResult *result
 	defer func() { writeJson(w, jobResult) }()
 
-	// use GetAllData to get all jobs
-	if datum, err := s.db.GetAllData(); err != nil {
+	if allJobs, err := s.getAllJobs(); err != nil {
 		log.Warnf("when list jobs, get all data failed: %+v", err)
 
 		jobResult = &result{
 			defaultResult: newErrorResult(err.Error()),
 		}
 	} else {
-		allJobs := make([]string, 0)
-		for _, eachJob := range datum["jobs"] {
-			allJobs = append(allJobs, strings.Trim(strings.Split(eachJob, ",")[0], " "))
-		}
-
 		jobResult = &result{
 			defaultResult: newSuccessResult(),
 			Jobs:          allJobs,
 		}
 	}
 }
 
+// show all jobs state
+func (s *HttpService) showJobStateHandler(w http.ResponseWriter, r *http.Request) {
+	log.Infof("show job state")
+
+	type result struct {
+		*defaultResult
+		Data string `json:"data,omitempty"`
+	}
+	var showJobStateResult *result
+	defer func() { writeJson(w, showJobStateResult) }()
+
```

Comment:
```suggestion
	defer func() { writeJson(w, showJobStateResult) }()

```

Write plain/text instead of json.